docs/themes/oso-webpack/index.js

```diff
@@ -28,6 +28,10 @@ import('monaco-editor-core').then(monaco => {
 
   monaco.languages.setMonarchTokensProvider('polar', {
     keywords: [
+      'type',
+      // TODO: want to make these keywords, but then variables named 'resource' are highlighted
```

Comment:
https://www.npmjs.com/package/monaco-editor-textmate Maybe worth a look?

Reply:
(so we can reuse the existing syntax highlighting definition)

Reply:
Yeah that's a good idea, follow up to this though.

Reply:
(Task in this cycle if we get to it)